### DISCUSSION TITLE
Implementing IFCOpenShell in a Supabase-React App Possibility?

### DISCUSSION THREAD

**Discussion by akhil-dignizant** [IfcOpenShell/IfcOpenShell#5106] [General]
I'm planning to make an app using React and Supabase as the backend. I have to implement IFCOpenShell, which typically works in a server environment using Python. In Supabase, the only way to handle server logic is through Edge Functions powered by a Deno server, which uses a JavaScript environment. However, IFCOpenShell supports only Python.

Is there any possibility of implementing IFCOpenShell in my Supabase-React app? How can I achieve this?

**Comment by aothms** [comment]
You can probably spawn a Python subprocess existing on the machine where the Deno server runs. A more adventurous route would be to attempt Web Assembly https://github.com/IfcOpenShell/IfcOpenShell/pull/2617
